assets/src/features/home-page/components/VideoroomHomePage.tsx

```diff
@@ -60,56 +50,67 @@ const VideoroomHomePage: React.FC = () => {
-            className="w-80 sm:w-96"
-          />
-          <Input
-            value={displayNameInput}
-            onChange={(event) => setDisplayNameInput(event.target.value)}
-            name="display_name"
-            type="text"
-            placeholder="Display name"
-            label="Your name"
-            className="w-80 sm:w-96"
-          />
-          <div className="space-y-1">
-            {checkboxes.map(({ label, id, status, onChange }) => (
-              <Checkbox key={id} label={label} id={id} status={status} onChange={onChange} />
-            ))}
+        <div className="flex max-h-[400px] w-full flex-col justify-between gap-x-24 gap-y-8 sm:flex-row 2xl:max-h-[500px]">
+          <div className="h-full w-full sm:h-[400px] sm:max-w-[600px] 2xl:h-[500px] 2xl:w-[750px] 2xl:max-w-none">
+            <HomePageVideoTile displayName={displayNameInput} />
           </div>
-          <div className="flex w-full items-center justify-center">
-            <Button
-              onClick={() => {
-                localStorage.setItem("displayName", displayNameInput);
-                setUsername(displayNameInput);
-                simulcast.setSimulcast(simulcastInput);
-                manualMode.setManualMode(manualModeInput);
-                cameraAutostart.setCameraAutostart(autostartCameraAndMicInput);
-              }}
-              href={`/room/${roomIdInput}`}
-              name="join"
-              variant="normal"
-              disabled={buttonDisabled}
-            >
-              Join the room
-            </Button>
+          <div className={clsx("flex flex-col items-center justify-center", roomId ? "gap-y-12" : "gap-y-6")}>
+            {roomId ? (
+              <div className="flex w-full flex-col items-center justify-center text-center">
+                <span>You are joining:</span>
+                <span className="text-2xl font-medium">{roomId}</span>
+              </div>
+            ) : (
+              <Input
+                value={roomIdInput}
+                onChange={(event) => setRoomIdInput(event.target.value)}
+                type="text"
+                required
+                name="room_name"
+                placeholder="Room name"
+                label="Room name"
+                disabled={!!roomId}
+                className="w-80 sm:w-96"
+              />
+            )}
+            <Input
+              value={displayNameInput}
+              onChange={(event) => setDisplayNameInput(event.target.value)}
+              name="display_name"
+              type="text"
+              placeholder="Display name"
+              label="Your name"
+              className="w-80 sm:w-96"
+            />
+            <div className="hidden space-y-1 sm:block">
+              {checkboxes.map(({ label, id, status, onChange }) => (
+                <Checkbox key={id} label={label} id={id} status={status} onChange={onChange} />
+              ))}
+            </div>
```

Comment:
The checkboxes here are not aligned 
![image](https://user-images.githubusercontent.com/2119556/217902438-0959e7cd-4df3-4e5d-945b-6f42cbf5b0f1.png)